src/main/java/com/dyescape/dyescapebot/configuration/moderation/ModerationConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package com.dyescape.dyescapebot.configuration.moderation;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Class used for moderation specific configuration settings.
+ * @author Dennis van der Veeke
+ * @since 0.1.0
+ */
+@Configuration
+@ConfigurationProperties(prefix = "moderation")
+public class ModerationConfiguration {
+
+    private long warningperiod;
+
+    /**
+     * Return a long, representing seconds, indicating how long a given warning should remain active.
+     * @return Warning time
+     * @author Dennis van der Veeke
+     * @since 0.1.0
+     */
+    public long getWarningperiod() {
```

Comment:
CamelCase => WarningPeriod

Reply:
Yeah, typo. Good catch.

Reply:
Oh, I remember. Spring explicitly wants this naming convention on the functions, otherwise it changes how the configuration option is noted. If we make it camelcase, it would expect a dash in the configuration options from what I see